core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -49,7 +56,7 @@ object RawlsApiService {
     import DefaultJsonProtocol._
     RejectionHandler.default.mapRejectionResponse {
       case res @ HttpResponse(status, _, ent: HttpEntity.Strict, _) =>
-        res.withEntity(entity = HttpEntity(ContentTypes.`application/json`, Map(status.toString -> ent.data.utf8String).toJson.toString))
+        res.withEntity(entity = HttpEntity(ContentTypes.`application/json`, Map(status.toString -> ent.data.utf8String).toJson.prettyPrint))
```

Comment:
not technically necessary for this PR. The default for `toString` is `compactPrint`, which can be hard for humans to read; this makes it easier for humans.